.github/workflows/CI.yml

```diff
@@ -13,7 +13,8 @@ jobs:
       matrix:
         julia-version: ['1.8']
         julia-arch: [x64]
-        os: [ubuntu-latest, windows-latest, macOS-11]
+        # os: [ubuntu-latest, windows-latest, macOS-11]
+        os: [windows-latest, macOS-11]
```

Comment:
@zolanaj do we definitely want to keep `ubuntu-latest` out of the test sets from here on out? Did you test with ubuntu with these latest changes?

Reply:
This is temporary, I believe we will bring it back soon.  @Bill-Becker may have more to say on this.